src/Packetery/Core/CoreHelper.php

```diff
@@ -55,15 +55,17 @@ public static function simplifyFloat( ?float $value, int $maxDecimalPlaces ): ?f
 	 * Trims the decimals to a desired format.
 	 *
 	 * @param float $value Value.
-	 * @param int   $position Position of a decimal.
+	 * @param int   $decimals Position of a decimal.
 	 *
 	 * @return string
 	 */
-	public static function trimDecimalPlaces( float $value, int $position ): string {
-		$formattedValue = number_format( $value, $position, '.', '' );
+	public static function trimDecimalPlaces( float $value, int $decimals ): string {
```

Comment:
a btw. proč to je vůbec v CoreHelperu, když se to používá v Modelu? Je v tom nějaký bordel se mi zdá. Není mi úplně jasné jaký je rozdíl mezi těmito Helpery.

A ještě bych zvážil, jestli to nepřejmenovat tu funkci, protože na pozadí ten number formát i zaokrohluje, všiml jsem si toho už když jsem to testoval viz [zde](https://www.php.net/manual/en/function.number-format.php#88424). Tak ať nás to pak někdy nevydrbe

Reply:
Bylo vysvetleno na standupu.